src/Command.php

```diff
@@ -40,6 +43,22 @@ public function insertWithReturningPks(string $table, array $columns): bool|arra
         return $result;
     }
 
+    protected function queryInternal(int $queryMode): mixed
+    {
+        try {
+            return parent::queryInternal($queryMode);
+        } catch (IntegrityException $e) {
```

Comment:
I wonder why it's `IntegrityException`...

Reply:
It is converted here https://github.com/yiisoft/db/blob/6a8ceea2395c5da253c62773e2bfd360c424be51/src/Exception/ConvertException.php#L41